docs/learn-glossary.md

```diff
@@ -3,160 +3,175 @@ id: learn-glossary
 
-### Run Id
+### Run ID
 
-A UUID that a Temporal service assigns to each workflow run. If allowed by
-a configured policy, you might be able to re-execute a workflow, after it has
-closed or failed, with the same _Workflow Id_. Each such re-execution is called
-a run. _Run Id_ is used to uniquely identify a run even if it shares a _Workflow Id_
-with others.
+A Run ID is UUID that a Temporal service assigns to each Workflow run.
+
+- If allowed by a configured policy, you might be able to re-execute a Workflow, after it has closed or failed, with the same [Workflow ID](#workflow-id).
+- Each such re-execution is called a run. Run ID is used to uniquely identify a run even if it shares a Workflow ID with others.
 
 ### Signal
 
-An external asynchronous request to a workflow. It can be used to deliver
-notifications or updates to a running workflow at any point in its existence.
+A Signal is an external asynchronous request to a [Workflow](#workflow).
+
+- A Signal can be used to deliver notifications or updates to a running [Workflow](#workflow) at any point in its existence.
 
 ### Task
 
-The context needed to execute a specific activity or workflow state transition.
-There are two types of tasks: an [Activity task](#activity-task) and a [Decision task](#decision-task)
-(aka workflow task). Note that a single activity execution corresponds to a single activity task,
-while a workflow execution employs multiple decision tasks.
+A Task is the context needed to execute a specific [Activity](#activity) or Workflow state transition.
+
+- There are two types of tasks:
+  1. [Activity Task](#activity-task)
+  2. [Workflow Task](#workflow-task)
+- A single [Activity](#activity) execution corresponds to a single [Activity Task](#activity-task), while a [Workflow Execution](#workflow-execution) employs multiple [Workflow Tasks](#workflow-task).
 
 ### Task Queue
 
-Common name for [activity task queues](#activity-task-queue) and [decision task queues](#decision-task-queue)
+Common name for [Activity Task Queues](#activity-task-queue) and [Workflow Task Queues](#workflow-task-queue)
 
 ### Task Token
 
-A unique correlation Id for a Temporal activity. Activity completion calls take either task token
-or Namespace, WorkflowId, ActivityId arguments.
+A Task Token is a unique correlation ID for a Temporal [Activity](#activity).
+
+- [Activity](#activity) completion calls take a Task Token, [Namespace](#namespace), [Workflow ID](#workflow-id), or [Activity ID](#activity-id) as an argument.
 
 ### Worker
 
-Also known as a _worker service_. A service that hosts the workflow and
-activity implementations. The worker polls the Temporal service for tasks, performs
-those tasks, and communicates task execution results back to the Temporal service.
-Worker services are developed, deployed, and operated by Temporal customers.
+A Worker, also known as a Worker service, is a service that hosts the [Workflow](#workflow) and [Activity](#activity) implementations.
```

Comment:
We don't use worker service anywhere in the code. I think we should remove the mention of it.

Reply:
Worker contains both ActivityWorker and WorkflowWorker